Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanelModifier.swift

```diff
@@ -89,12 +99,15 @@ private struct FloatingPanelModifier<PanelContent>: ViewModifier where PanelCont
         content
             .overlay(alignment: Alignment(horizontal: horizontalAlignment, vertical: .top)) {
                 FloatingPanel(
+                    attributionBarHeight: attributionBarHeight,
                     backgroundColor: backgroundColor,
                     selectedDetent: selectedDetent,
                     isPresented: isPresented,
                     content: panelContent
                 )
-                .ignoresSafeArea(.all, edges: .bottom)
+                // When the panel is anchored to the bottom of the screen (compact) ignore the
+                // device's bottom safe area.
+                .ignoresSafeArea(.container, edges: isCompact ? .bottom : [])
```

Comment:
We had essentially opted out of the system provided keyboard avoidance by using `.all` here.